dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskExecutionContext.java

```diff
@@ -218,7 +218,8 @@ public class TaskExecutionContext implements Serializable {
     private ExecutionStatus currentExecutionStatus;
 
     /**
-     *  Task Logger name should be like: Task-{processDefinitionId}-{processInstanceId}-{taskInstanceId}
+     * Task Logger name should be like:
+     * TaskAppId=TASK-{firstSubmitTime}-{processDefineCode}_{processDefineVersion}-{processInstanceId}-{taskInstanceId}
```

Comment:
Why did we need to add `TaskAppId=` in the prefix? 
I think the `TASK-{firstSubmitTime}-{processDefineCode}_{processDefineVersion}-{processInstanceId}-{taskInstanceId}` is enough
cc @caishunfeng